modules/collections/client/config/collections.routes.js

```diff
@@ -72,6 +72,7 @@ angular.module('collections').config(['$stateProvider', function($stateProvider)
           }
           CollectionModel.add($scope.collection)
             .then(function(/* model */) {
+              AlertService.success('"' + $scope.collection.displayName + '" created successfully.', 4000, true);
```

Comment:
It looks like there was no alert here before, do we want one now?

Reply:
Personally, I think it increases usability in the old interface. Not that it will matter if they sell the new interface to the stakeholders.